internal/services/datafactory/data_factory_linked_service_azure_postgresql.go

```diff
@@ -0,0 +1,315 @@
+// Copyright (c) HashiCorp, Inc.
+// SPDX-License-Identifier: MPL-2.0
+
+package datafactory
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/hashicorp/go-azure-sdk/resource-manager/datafactory/2018-06-01/factories"
+	"github.com/hashicorp/terraform-provider-azurerm/helpers/tf"
+	"github.com/hashicorp/terraform-provider-azurerm/internal/clients"
+	"github.com/hashicorp/terraform-provider-azurerm/internal/services/datafactory/parse"
+	"github.com/hashicorp/terraform-provider-azurerm/internal/services/datafactory/validate"
+	"github.com/hashicorp/terraform-provider-azurerm/internal/tf/pluginsdk"
+	"github.com/hashicorp/terraform-provider-azurerm/internal/tf/validation"
+	"github.com/hashicorp/terraform-provider-azurerm/internal/timeouts"
+	"github.com/hashicorp/terraform-provider-azurerm/utils"
+	"github.com/tombuildsstuff/kermit/sdk/datafactory/2018-06-01/datafactory"
+)
+
+func resourceDataFactoryLinkedServiceAzurePostgreSQL() *pluginsdk.Resource {
+	return &pluginsdk.Resource{
+		Create: resourceDataFactoryLinkedServiceAzurePostreSQLCreateUpdate,
```

Comment:
We're requiring new resources to have split create/update methods as having them together can have unintended consequences. We're slowly migrating every existing resource to use this format so we'd like new resources to come in this way as well.